charts/victoria-metrics-cluster/values.yaml

```diff
@@ -168,6 +168,8 @@ vmselect:
     clusterIP: ""
     # -- Service External IPs. Ref: [https://kubernetes.io/docs/user-guide/services/#external-ips](https://kubernetes.io/docs/user-guide/services/#external-ips)
     externalIPs: []
+    # -- Extra service ports
+    extraServicePorts: []
```

Comment:
Can you also add extraServicePorts to the vmstorage definition for consistency?

Reply:
Done!